TCAT/Controllers/HomeViewController.swift

```diff
@@ -46,7 +47,8 @@ class HomeViewController: UIViewController {
     var loadingIndicator: LoadingIndicator?
     var isLoading: Bool { return loadingIndicator != nil }
 
-    let reachability = Reachability(hostname: Network.ipAddress)
+    let reachability = Reachability(hostname: Endpoint.config.host ?? "")
```

Comment:
Does FutureNova define this as optional? No way around this?

Reply:
yea its an optional from futurenova